CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ Swift 3
 * Curried function syntax has been removed, and now produces a compile-time
   error.
 
+* Generic signatures can now contain superclass requirements with dependent types,
```

Comment:
"Dependent types" might give people the wrong impression. 'Superclass requirements involving generic parameters' should be clearer.